packages/popup/src/App.js

```diff
@@ -76,102 +283,26 @@ function App() {
-                exact
-                path={HOME}
-                component={HomePage}
-              />
-              <Route exact path={CURRENT_SEED_PHRASE} component={CurrentSeed} />
-              <Route exact path={NEW_SEED_PHRASE} component={NewSeed} />
-              <Route exact path={BACKUP_SEED_PHRASE} component={BackupSeed} />
-              <Route exact path={CONFIRM_SEED_PHRASE} component={ConfirmSeed} />
-              <Route
-                exact
-                path={IMPORT_SEED_PHRASE}
-                component={ImportSeedPhrase}
-              />
-              <Route
-                exact
-                path={WALLET_IMPORT_PRIVATE_KEY}
-                component={ImportPrivateKey}
-              />
-              <Route
-                exact
-                path={SEND_TRANSACTION}
-                component={SendTransaction}
-              />
-              <Route
-                exact
-                path={CONFIRM_TRANSACTION}
-                component={ConfirmTransaction}
-              />
-              <Route exact path={EDIT_GAS_FEE} component={EditGasFee} />
-              <Route exact path={EDIT_PERMISSION} component={EditPermission} />
-              <Route exact path={SET_PASSWORD} component={SetPassword} />
-              <Route
-                exact
-                path={SELECT_CREATE_TYPE}
-                component={SelectCreateType}
-              />
-              <Route exact path={VIEW_DATA} component={ViewData} />
-              <Route exact path={CONNECT_SITE} component={ConnectSite} />
-              <Route
-                exact
-                path={CONFIRM_ADD_SUGGESTED_TOKEN}
-                component={ConfirmAddSuggestedToken}
-              />
-              <Route
-                exact
-                path={REQUEST_SIGNATURE}
-                component={RequestSignature}
-              />
-              <Route
-                exact
-                path={DAPP_SWITCH_NETWORK}
-                component={DappSwitchNetwork}
-              />
-              <Route exact path={DAPP_ADD_NETWORK} component={DappAddNetwork} />
-              <Route exact path={HISTORY} component={History} />
-              <Route
-                exact
-                path={ACCOUNT_MANAGEMENT}
-                component={AccountManagement}
-              />
-              <Route exact path={EXPORT_SEED} component={ExportSeed} />
-              <Route
-                exact
-                path={EXPORT_PRIVATEKEY}
-                component={ExportPrivateKey}
-              />
-              <Route exact path={ERROR} component={ErrorPage} />
-              <Route path="*" render={() => <Redirect to={ERROR} />} />
-            </Switch>
+    <ErrorBoundary
+      FallbackComponent={ErrorPage}
+      onError={error => setFatalError(error)}
+    >
+      <Suspense fallback={<PageLoading />}>
+        <Router>
+          <div className="h-150 w-93 m-auto light relative overflow-hidden">
```

Comment:
there's `useQuery` in `ErrorPage` which requires it to be under `<Router>`

Reply:
`Router` need be placed out of `ErrorBoundary` with the same reason